### DISCUSSION TITLE
Visual Glitch When Maximising Custom Window

### DISCUSSION THREAD

**Discussion by tywil04** [AvaloniaUI/Avalonia#7471] [Q&A] [answered]
Hey, I am new to Avalonia I chose it because I wanted to create applications that worked both on Linux and Windows. I wanted the apps to look the same on both platforms and for this I decided to create everything from scratch. For the most part, I have done this however I am having an issue with window maximization - whenever I maximize my window it glitches into place, it doesn't play the same animation as other native windows.

Here is a video of the issue at hand:
https://user-images.githubusercontent.com/68185249/151684899-e3754d32-0c28-4006-b0bf-24adb83708b1.mp4

To maximise the window I am using a function that contains: `this.WindowState = WindowState.Maximised;`

The issue I believe was present on Linux, however I cannot confirm that right now as my Linux install died on me last night.

I have tried everything to fix this but so far nothing has worked - some help would be much appreciated!

**Comment by tywil04** [comment, marked as answer]
Nevermind - I am I moron I just solved the issue by changing some properties of `<Window/>`

**Reply by timunie** [reply]
May you share your solution if others face the same? Which Properties did you modify?

**Reply by tywil04** [reply]
Sure! Here are the properties I changed and what I set them to:
`ExtendClientAreaToDecorationsHint='True'`
`SystemDecorations='Full'`
`ExtendClientAreaChromeHints='NoChrome'`
`ExtendClientAreaTitleBarHeightHint='-1'`
